release/summarize_pr.sh

```diff
@@ -0,0 +1,24 @@
+#
+# Summarize pull requests.  Example usage:
+#
+#   bash summarize_pr.sh 123 456 789
+#
+# where 123, 456 and 789 are PR IDs.  List as many PRs as you want.
+#
+# Gives a summary like:
+#
+# - Removes native xz support (PR [#282](https://github.com/RaRe-Technologies/smart_open/pull/282), [@tdhopper](https://github.com/tdhopper))
+# - backward compatibility fixes (PR [#294](https://github.com/RaRe-Technologies/smart_open/pull/294), [@mpenkov](https://github.com/mpenkov))
+#
+# You can paste this directly into CHANGELOG.md
+#
+for prid in "$@"
+do
+    api_url="https://api.github.com/repos/RaRe-Technologies/smart_open/pulls/${prid})"
+    json="$(wget --quiet -O - ${api_url})"
+    title="$(echo "$json" | jq .title --raw-output)"
```

Comment:
Is this installed by default? Do the scripts assume any specific environment / OS / dependencies?

Reply:
Good catch. I'll document the dependencies. To answer your questions briefly: 

- no, but wget/json/etc packages are easily available for most Linux distros
- There are dependencies like bash and the above utils, but they are really general

Reply:
Thanks. I can also test this on my environment (OSX, GNU bash).